test/factories/feed_factory.rb

```diff
@@ -14,6 +14,12 @@
     enclosure_template { Feed.enclosure_template_default }
     enclosure_prefix { "" }
 
+    after(:create) do |feed, evaluator|
```

Comment:
a handful of tests in a couple different models started failing after removing the after create hook in the Feed model; i think because feeds created in this way don't pass through the controller, but they assume episodes are included in non-default feeds. i added this hook into the factory as one solution to that, though upon reflection it might be better to just go into those individual suites and call `feed.set_default_episodes` in those files specifically?